kinds/apis/v1beta1/runtimeconfig_types.go

```diff
@@ -26,6 +26,12 @@ type RuntimeConfigSpec struct {
 	AdminConsole AdminConsoleSpec `json:"adminConsole,omitempty"`
 	// LocalArtifactMirrorPort holds the Local Artifact Mirror configuration.
 	LocalArtifactMirror LocalArtifactMirrorSpec `json:"localArtifactMirror,omitempty"`
+
+	// Proxy holds the proxy configuration.
+	Proxy *ProxySpec `json:"proxy,omitempty"`
```

Comment:
Ultimately this does belong here but I think a refactor must first be done by our team

Reply:
ack